src/content/blog/whats-new-februrary-2025.mdx

```diff
@@ -0,0 +1,220 @@
+---
+title: "What's new in Astro - February 2025"
+description: "February 2025 - Storyblok live previews, Astro Integrations at TheJam.dev, behind the scenes at Astro Together, and more!"
+publishDate: "February 28, 2025"
+authors:
+  - sarah
+coverImage: "/src/content/blog/_images/whats-new-shared.webp"
+socialImage: "/src/content/blog/_images/whats-new-february/og.webp"
+lang: "en"
+startDate: "2025-02-01"
+endDate: "2025-03-01"
+---
+import MonthlyShowcaseGrid from './_whats-new-components/MonthlyShowcaseGrid.astro';
+import ShowcaseGridById from './_whats-new-components/ShowcaseGridById.astro';
+import ThemeGrid from './_whats-new-components/ThemeGrid.astro';
+import YouTubeGrid from './_whats-new-components/YouTubeGrid.astro';
+import { Tweet } from 'astro-embed';
+
+Some exciting news from around the Astro ecosystem this month, including:
+
+- [Updates from Mission Control](#updates-from-mission-control)
+- [Latest releases](#latest-releases)
+- [Astro Adopters](#astro-adopters)
+- [Weird Wide Web](#weird-wide-web)
+- [Community updates](#community)
+- [Astro in the news, migration stories, how-to articles, videos, and more!](#content)
+- [Tips & Tools](#tips--tools)
+- [Themes & Templates](#themes--templates)
+- [Showcase](showcase)
+- [Starlight in the wild](#starlight-in-the-wild)
+
+Let's dive right into it!
+
+## Updates from Mission Control 
+
+- 🅱 [Stable Live Previews for Storyblok’s Astro SDK](https://www.storyblok.com/mp/announcing-stable-live-preview-for-storybloks-astro-sdk)
+- 🇨🇦 [Case study: How Vocation Events brought tech company Astro together](https://blog.mtl.org/en/case-study-vocation-events)
+- 🍓 [Astro Integrations with Florian Lefebvre at TheJam.dev](https://cfe.dev/sessions/jamdev2025-astro-integrations/)
+
+## Latest Releases
+
+Minor releases introduce new features and improvements. Check out what's new!
+
+- [Astro 5.3](https://github.com/withastro/astro/blob/refs/heads/main/packages/astro/CHANGELOG.md#530) - faster page rendering, easier setup for experimental sessions, and more!
+- [Astro 5.4](https://github.com/withastro/astro/blob/refs/heads/main/packages/astro/CHANGELOG.md#540) - remote image optimization in Markdown, enhanced security for dev and preview servers, and more!
+- [Starlight 0.32](https://github.com/withastro/starlight/blob/main/packages/starlight/CHANGELOG.md#0310) - New route data middleware, i18n API access for plugins, and more!
+ 
+## Astro Adopters
+
+From Porche to IKEA to Jamie Oliver, some of the biggest names you know are using Astro!
```

Comment:
Should this mention instead "From Michelin to NordVPN" or something like that?

Reply:
I was thinking that this would be standard wording that wouldn't change, just we'd update whenever we spot a new big site.

Does that feel confusing?

Reply:
Thought about that but was not sure. I wouldn't say confusing, but enough to make me think about it. Could be worth checking what other people think considering I'm not a native speaker?

Reply:
I see your point! Easy enough to change. It's a brand new section anyway. There are no rules!!